ptychodus/model/ptychonn/model.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import logging
+
+import lightning
+import ptychonn
```

Comment:
Both of these imports cause ptychonn to become a dependency for the base ptychodus package. I'm not sure how to avoid these imports, but ptychonn should not be a requirement to run ptychodus.